Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node
+
+WORKDIR /app
+
+ADD . /app
```

Comment:
Specifying `/app` (the WORKDIR) here again is a voilation of [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
Fixed.

Reply:
would I have `COPY . WORKDIR` instead?